[
{"id":null,"parent_id":null,"created":"\/Date(1632853907765)\/","ws_group":"PatchBusinessPartner","action":"Update Business Partner","visual_aid_label":null,"visual_aid_value":null,"input":"{\"oDataBP\":\"{\\\"CardCode\\\": \\\"AAA00006\\\",\\\"VatIDNum\\\": 123, \\\"CardName\\\": \\\"Prueba\\\", \\\"CardType\\\": \\\"cCustomer\\\",\\\"Address\\\": \\\"Novitec\\\",\\\"ContactPerson\\\": null,\\\"EmailAddress\\\": \\\"prueba@gmail.it\\\",\\\"Phone1\\\": \\\"095408122\\\",\\\"GroupCode\\\": 104,\\\"U_Branch\\\": \\\"IT\\\",\\\"U_ReasonCustomer\\\": null,\\\"Currency\\\": \\\"EUR\\\",\\\"PayTermsGrpCode\\\": 3,\\\"PriceListNum\\\": 1,\\\"U_SurgeonAfiliation\\\": null,\\\"BPAddresses\\\": [{\\\"AddressType\\\": \\\"bo_BillTo\\\",\\\"AddressName\\\": \\\"Prueba\\\",\\\"City\\\": \\\"CATANIA\\\",\\\"ZipCode\\\": \\\"95126\\\",\\\"Country\\\": \\\"IT\\\"},{\\\"AddressType\\\": \\\"bo_ShipTo\\\",\\\"AddressName\\\": \\\"Prueba\\\",\\\"City\\\": \\\"CATANIA\\\",\\\"ZipCode\\\": \\\"95126\\\",\\\"Country\\\": \\\"IT\\\"}]}\"}","old_input":null,"output":"{\"success\":false,\"error\":\"Ocurrio un error al actualizar un Business partner\"}","nvt_code":5004,"nvt_message":"NVTException 5004: Novitec.Mupet.Common.NVTException:  The underlying connection was closed: A connection that was expected to be kept alive was closed by the server. ---\u003e System.Net.WebException: The underlying connection was closed: A connection that was expected to be kept alive was closed by the server.\r\n   at MupetJoy.Security.ClienteRestBLSAP.LeerStatus(IRestResponse response) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\Security\\ClienteRestBLSAP.cs:line 162\r\n   at MupetJoy.Security.ClienteRestBLSAP.LeerUpdate(IRestResponse response) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\Security\\ClienteRestBLSAP.cs:line 155\r\n   at MupetJoy.Security.ClienteRestBLSAP.EjecutarPatch(String link, String parametro) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\Security\\ClienteRestBLSAP.cs:line 70\r\n   at MupetJoy.BLL.BusinessPartner_BLL.UpdateBusinessPartner(String oDataBP) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\BLL\\BusinessPartner_BLL.cs:line 40\r\n   at MupetJoy.Controllers.BusinessPartnerController.\u003c\u003ec__DisplayClass2_0.\u003cPatchBusinessPartner\u003eb__0(SimpleProcess scenario) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\Controllers\\BusinessPartnerController.cs:line 225\r\n   at Novitec.Mupet.Scenarios.SimpleProcess.ProcessBody()\r\n   at Novitec.Mupet.Scenarios.Scenario`1.\u003c\u003ec__DisplayClass48_0.\u003cFinish\u003eb__0()\r\n   at Novitec.Mupet.Scenarios.Scenario`1.ProcessCatchable(Action action)\r\n   --- End of inner exception stack trace ---","sap_code":null,"sap_message":null,"sap_hook":null,"thirdparty_hook":null,"status":"E","finished":"\/Date(1632853961352)\/","log_error":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---\u003e System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteInsertIdentity(String query, Dictionary`2 parameters, String idColumn)\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteInsert(Query query, Dictionary`2 parameters)\r\n   at Novitec.Mupet.Common.LogEntryModel.StartLogEntryDB()\r\n   at Novitec.Mupet.Common.LogEntryModel.StartLogEntry(Nullable`1 parentId)\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20\r\nSystem.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---\u003e System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Novitec.Mupet.SAP.DBAccessSAP..ctor()\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteNonQuery(String query, Dictionary`2 parameters)\r\n   at Novitec.Mupet.Common.LogEntryModel.EndLogEntryDB()\r\n   at Novitec.Mupet.Common.LogEntryModel.EndLogEntry(String output, String status, NVTException nvtException, SAPException sapException)\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20"},
{"id":null,"parent_id":null,"created":"\/Date(1632855413669)\/","ws_group":"PatchBusinessPartner","action":"Update Business Partner","visual_aid_label":null,"visual_aid_value":null,"input":"{\"oDataBP\":\"{\\\"CardCode\\\": \\\"AAA00006\\\",\\\"VatIDNum\\\": 123, \\\"CardName\\\": \\\"Prueba\\\", \\\"CardType\\\": \\\"cCustomer\\\",\\\"Address\\\": \\\"Novitec\\\",\\\"ContactPerson\\\": null,\\\"EmailAddress\\\": \\\"prueba@gmail.it\\\",\\\"Phone1\\\": \\\"095408122\\\",\\\"GroupCode\\\": 104,\\\"U_Branch\\\": \\\"IT\\\",\\\"U_ReasonCustomer\\\": null,\\\"Currency\\\": \\\"EUR\\\",\\\"PayTermsGrpCode\\\": 3,\\\"PriceListNum\\\": 1,\\\"U_SurgeonAfiliation\\\": null,\\\"BPAddresses\\\": [{\\\"AddressType\\\": \\\"bo_BillTo\\\",\\\"AddressName\\\": \\\"Prueba\\\",\\\"City\\\": \\\"CATANIA\\\",\\\"ZipCode\\\": \\\"95126\\\",\\\"Country\\\": \\\"IT\\\"},{\\\"AddressType\\\": \\\"bo_ShipTo\\\",\\\"AddressName\\\": \\\"Prueba\\\",\\\"City\\\": \\\"CATANIA\\\",\\\"ZipCode\\\": \\\"95126\\\",\\\"Country\\\": \\\"IT\\\"}]}\"}","old_input":null,"output":"{\"success\":false,\"error\":\"Ocurrio un error al actualizar un Business partner\"}","nvt_code":5004,"nvt_message":"NVTException 5004: Novitec.Mupet.Common.NVTException:  Unexpected character encountered while parsing value: \u003c. Path \u0027\u0027, line 0, position 0. ---\u003e Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: \u003c. Path \u0027\u0027, line 0, position 0.\r\n   at Newtonsoft.Json.JsonTextReader.ParseValue()\r\n   at Newtonsoft.Json.JsonTextReader.Read()\r\n   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)\r\n   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value)\r\n   at MupetJoy.Controllers.BusinessPartnerController.\u003c\u003ec__DisplayClass2_0.\u003cPatchBusinessPartner\u003eb__0(SimpleProcess scenario) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\Controllers\\BusinessPartnerController.cs:line 226\r\n   at Novitec.Mupet.Scenarios.SimpleProcess.ProcessBody()\r\n   at Novitec.Mupet.Scenarios.Scenario`1.\u003c\u003ec__DisplayClass48_0.\u003cFinish\u003eb__0()\r\n   at Novitec.Mupet.Scenarios.Scenario`1.ProcessCatchable(Action action)\r\n   --- End of inner exception stack trace ---","sap_code":null,"sap_message":null,"sap_hook":null,"thirdparty_hook":null,"status":"E","finished":"\/Date(1632855693896)\/","log_error":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---\u003e System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteInsertIdentity(String query, Dictionary`2 parameters, String idColumn)\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteInsert(Query query, Dictionary`2 parameters)\r\n   at Novitec.Mupet.Common.LogEntryModel.StartLogEntryDB()\r\n   at Novitec.Mupet.Common.LogEntryModel.StartLogEntry(Nullable`1 parentId)\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20\r\nSystem.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---\u003e System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Novitec.Mupet.SAP.DBAccessSAP..ctor()\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteNonQuery(String query, Dictionary`2 parameters)\r\n   at Novitec.Mupet.Common.LogEntryModel.EndLogEntryDB()\r\n   at Novitec.Mupet.Common.LogEntryModel.EndLogEntry(String output, String status, NVTException nvtException, SAPException sapException)\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20"},
{"id":null,"parent_id":null,"created":"\/Date(1632855949241)\/","ws_group":"PatchBusinessPartner","action":"Update Business Partner","visual_aid_label":null,"visual_aid_value":null,"input":"{\"oDataBP\":\"{\\\"CardCode\\\": \\\"AAA00006\\\",\\\"VatIDNum\\\": 123, \\\"CardName\\\": \\\"Prueba\\\", \\\"CardType\\\": \\\"cCustomer\\\",\\\"Address\\\": \\\"Novitec\\\",\\\"ContactPerson\\\": null,\\\"EmailAddress\\\": \\\"prueba@gmail.it\\\",\\\"Phone1\\\": \\\"095408122\\\",\\\"GroupCode\\\": 104,\\\"U_Branch\\\": \\\"IT\\\",\\\"U_ReasonCustomer\\\": null,\\\"Currency\\\": \\\"EUR\\\",\\\"PayTermsGrpCode\\\": 3,\\\"PriceListNum\\\": 1,\\\"U_SurgeonAfiliation\\\": null,\\\"BPAddresses\\\": [{\\\"AddressType\\\": \\\"bo_BillTo\\\",\\\"AddressName\\\": \\\"Prueba\\\",\\\"City\\\": \\\"CATANIA\\\",\\\"ZipCode\\\": \\\"95126\\\",\\\"Country\\\": \\\"IT\\\"},{\\\"AddressType\\\": \\\"bo_ShipTo\\\",\\\"AddressName\\\": \\\"Prueba\\\",\\\"City\\\": \\\"CATANIA\\\",\\\"ZipCode\\\": \\\"95126\\\",\\\"Country\\\": \\\"IT\\\"}]}\"}","old_input":null,"output":"{\"success\":false,\"error\":\"Ocurrio un error al actualizar un Business partner\"}","nvt_code":5004,"nvt_message":"NVTException 5004: Novitec.Mupet.Common.NVTException:  Unexpected character encountered while parsing value: \u003c. Path \u0027\u0027, line 0, position 0. ---\u003e Newtonsoft.Json.JsonReaderException: Unexpected character encountered while parsing value: \u003c. Path \u0027\u0027, line 0, position 0.\r\n   at Newtonsoft.Json.JsonTextReader.ParseValue()\r\n   at Newtonsoft.Json.JsonTextReader.Read()\r\n   at Newtonsoft.Json.JsonReader.ReadForType(JsonContract contract, Boolean hasConverter)\r\n   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)\r\n   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)\r\n   at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value)\r\n   at MupetJoy.Controllers.BusinessPartnerController.\u003c\u003ec__DisplayClass2_1.\u003cPatchBusinessPartner\u003eb__0(SimpleProcess scenario) in Y:\\ProyectosGIT\\EstablishmentLabs\\MupetJoy\\MupetJoy\\Controllers\\BusinessPartnerController.cs:line 228\r\n   at Novitec.Mupet.Scenarios.SimpleProcess.ProcessBody()\r\n   at Novitec.Mupet.Scenarios.Scenario`1.\u003c\u003ec__DisplayClass48_0.\u003cFinish\u003eb__0()\r\n   at Novitec.Mupet.Scenarios.Scenario`1.ProcessCatchable(Action action)\r\n   --- End of inner exception stack trace ---","sap_code":null,"sap_message":null,"sap_hook":null,"thirdparty_hook":null,"status":"E","finished":"\/Date(1632855999882)\/","log_error":"System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---\u003e System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteInsertIdentity(String query, Dictionary`2 parameters, String idColumn)\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteInsert(Query query, Dictionary`2 parameters)\r\n   at Novitec.Mupet.Common.LogEntryModel.StartLogEntryDB()\r\n   at Novitec.Mupet.Common.LogEntryModel.StartLogEntry(Nullable`1 parentId)\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20\r\nSystem.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server) ---\u003e System.ComponentModel.Win32Exception (0x80004005): The network path was not found\r\n   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling, SqlAuthenticationProviderManager sqlAuthProviderManager)\r\n   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal\u0026 connection)\r\n   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.ProviderBase.DbConnectionClosed.TryOpenConnection(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)\r\n   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)\r\n   at System.Data.SqlClient.SqlConnection.Open()\r\n   at Novitec.Mupet.SAP.DBAccessSAP..ctor()\r\n   at Novitec.Mupet.Common.DBAccess.ExecuteNonQuery(String query, Dictionary`2 parameters)\r\n   at Novitec.Mupet.Common.LogEntryModel.EndLogEntryDB()\r\n   at Novitec.Mupet.Common.LogEntryModel.EndLogEntry(String output, String status, NVTException nvtException, SAPException sapException)\r\nClientConnectionId:00000000-0000-0000-0000-000000000000\r\nError Number:53,State:0,Class:20"},
